string arm
string target
float64 velocity_scaling
float64 acceleration_scaling
bool execute
---
bool success
string message
